src/app/query/stacks/balance/balance.query.ts

```diff
@@ -47,18 +48,19 @@ export function useAnchoredStacksAccountBalanceQuery<T extends unknown = FetchAc
   options?: AppUseQueryConfig<FetchAccountBalanceResp, T>
 ) {
   const client = useStacksClientAnchored();
+
   return useQuery({
     enabled: !!address,
     queryKey: ['get-address-anchored-stx-balance', address],
     queryFn: () => fetchAccountBalance(client)(address),
-    retryOnMount: true,
+
     ...balanceQueryOptions,
     ...options,
   });
 }
 
 export function useAnchoredAccountBalanceListQuery(accounts?: AccountWithAddress[]) {
-  const client = useStacksClient();
+  const client = useStacksClientAnchored();
```

Comment:
This is weird if causing bug now bc I looked back at my PR refactoring to the `StacksClient` and `BitcoinClient`, and this was using `useApi` (unanchored) before too.

Reply:
🤷🏼  I don't think this was causing the bug, but the hook is named achored and was using unachored